.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -62,22 +65,25 @@ body:
     validations:
       required: true
   - type: dropdown
-    id: sync
+    id: services
     attributes:
-      label: What SDK flavour are you using?
+      label: What Services are you using?
       options:
+        - -- select --
         - Local Database only
         - Atlas Device Sync
-        - Atlas App Services (auth, functions, etc.)
+        - 'Atlas App Services: Auth or Functions etc'
+        - Both Atlas Device Sync and Atlas App Services
```

Comment:
Is it really important to distinguish between Atlas App Services and Atlas Device Sync?  If you are using Atlas Device Sync, then you have to be using App Services (auth is required).  If someone is only using App Services without device sync, are they using Local Database then?

Reply:
It could be you are not using Device Sync, but just Functions. But perhaps we should leave out "Auth" as that confuses and just have "Function, GraphQL, DataAPI"?